attestation-agent/kbs_protocol/src/lib.rs

```diff
@@ -136,19 +136,16 @@ impl KbsProtocolWrapper {
 #[async_trait]
 impl KbsRequest for KbsProtocolWrapper {
     async fn http_get(&mut self, url_string: String) -> Result<Vec<u8>> {
-        let url = Url::parse(&url_string)
+        let _ = Url::parse(&url_string)
             .map_err(|e| anyhow!("Invalid Request URL {url_string}: {e}"))?;
 
-        // Use default port of KBS: 8080
-        let port = url.port().unwrap_or(8080);
-        let host_str = url.host_str().ok_or_else(|| anyhow!("No KBS host given"))?;
-        let kbs_url = format!("{}://{host_str}:{port}", url.scheme());
+        let root_url = url_string.split(KBS_URL_PREFIX).collect::<Vec<&str>>()[0].to_string();
```

Comment:
If the given URL does not have a `KBS_URL_PREFIX` const, is it still as expected? s.t.
```rust
let root_url = "google.com/not-kbs".split("/kbs/v0").collect::<Vec<&str>>();
// root_url is "google.com/not-kbs"
```

Reply:
This type of URL input without a `KBS_URL_PREFIX` const is illegal, that situation will not occur in the normal call scenario of this function